Source/WebCore/platform/graphics/gstreamer/ImageGStreamerSkia.cpp

```diff
@@ -47,16 +45,14 @@ ImageGStreamer::ImageGStreamer(GRefPtr<GstSample>&& sample)
     if (UNLIKELY(!GST_IS_BUFFER(buffer)))
         return;
 
-    GstVideoInfo videoInfo;
-    if (!gst_video_info_from_caps(&videoInfo, gst_sample_get_caps(m_sample.get())))
-        return;
-
-    auto videoFrame = makeUnique<GstMappedFrame>(buffer, &videoInfo, GST_MAP_READ);
+    auto videoFrame = makeUnique<GstMappedFrame>(m_sample, GST_MAP_READ);
```

Comment:
Not a big deal anymore but from what I see you could create this object in the heap.

Reply:
Done!